js/resource.js

```diff
@@ -56,6 +56,156 @@ $(document).ready(function(){
 		return false;
 	});
 
+	$(".showIssues").click(function () {
+	    $('.resource_tab_content').hide();
+		$('#div_issues').show();
+		$('#div_fullRightPanel').show();
+		updateIssues();
+		return false;
+	});
+
+	$(".issuesBtn").live("click", function(e) {
+		e.preventDefault();
+		getIssues($(this));
+	});
+
+	$(".downtimeBtn").live("click", function(e) {
+		e.preventDefault();
+		getDowntime($(this));
+	});
+
+	$("#submitCloseIssue").live("click", function() {
+		submitCloseIssue();
+	});
+
+	$("#submitNewIssue").live("click", function(e) {
+		e.preventDefault();
+		submitNewIssue();
+	});
+
+	$("#submitNewDowntime").live("click", function(e) {
+		e.preventDefault();
+		
+		var errors = [];
+
+		console.log($("#startDate").val());
+
+		if($("#startDate").val()=="") {	
+			errors.push({
+				message: "Must set a date.",
```

Comment:
Coral is now translatable. Please, in javascript files, put strings that will have to be translated between _() sub.

For example : _("Must set a date.")

In PHP, strings are found automatically if within HTML. If in a PHP function, you must also use _()